auditcare/management/commands/generate_request_report.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import re
+
+from django.contrib.auth.models import User
+from django.core.management.base import LabelCommand
+
+from corehq.apps.users.models import WebUser
+from dimagi.utils.couch.database import iter_docs
+
+from auditcare.models import NavigationEventAudit
+
+
+def navigation_event_ids_by_user(user):
+    database = NavigationEventAudit.get_db()
+
+    return {row['id'] for row in database.view('auditcare/urlpath_by_user_date',
+        startkey=[user ],
+        endkey=[user, {}],
+        reduce=False,
+        include_docs=False,
+    )}
+
+def filter_non_domain_request(domain, request_path):
+    return request_path.startswith('/a/' + domain)
```

Comment:
should there be a trailing slash after the domain? I think this would could `mvp-sada` domain access as requests made to the `mvp` domain, no?